src/Plugin/rest/resource/GetUserPreferences.php

```diff
@@ -0,0 +1,190 @@
+<?php
+namespace Drupal\ccf_user_preferences\Plugin\rest\resource;
+
+use Drupal\rest\Plugin\ResourceBase;
+use Drupal\user\Entity\User;
+use Drupal\paragraphs\Entity\Paragraph;
+use Drupal\rest\ModifiedResourceResponse;
+use Drupal\jwt\Transcoder\JwtTranscoderInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Drupal\ccf_user_preferences\Entity\CcfUserPreferences;
+
+/**
+ * Provides REST API for User preferences Details based
+ * on URL.
+ *
+ * @RestResource(
+ *   id = "get_user_preferences_rest_resource",
+ *   label = @Translation("Get User Preferences"),
+ *   uri_paths = {
+ *     "canonical" = "/api/get_user_preferences",
+ *     "create"    = "/api/add_user_preferences"
+ *   }
+ * )
+ */
+class GetUserPreferences extends ResourceBase
+{
```

Comment:
Run the phpcs with Drupal Coding standard. As per Drupal coding standard, the starting parenthesis should be on the same line. This mistake is at several places in this file.

Reply:
Updated phpcs. Pls check